packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -450,66 +450,65 @@
+                                data-ng-if="!hosting.sqlPriveInfo.hasPrivateDatabaseToActivate"
+                                data-ng-bind="privateDatabases.length"
+                            ></p>
+
                             <p
-                                data-ng-if="hosting.sqlPriveInfo.nbDataBaseInclude > hosting.sqlPriveInfo.nbDataBaseActive"
+                                data-ng-if="hosting.sqlPriveInfo.hasPrivateDatabaseToActivate"
                                 data-translate="hosting_dashboard_service_privatesql_active_remaining"
-                                data-translate-values="{
-                                   count: hosting.sqlPriveInfo.nbDataBaseInclude - hosting.sqlPriveInfo.nbDataBaseActive,
-                               }"
+                                data-translate-values="{ count: 1 }"
                             ></p>
-                            <oui-action-menu
-                                data-text="{{ ::'hosting_dashboard_service_privatesql_more' | translate }}"
-                                data-ng-if="privateDatabasesLinked.length > 0"
-                            >
-                                <div
-                                    data-ng-repeat="privateDb in privateDatabasesLinked track by privateDb.name"
-                                >
-                                    <oui-action-menu-item
-                                        data-text="{{::privateDb.name}}"
-                                        data-ng-if="!privateDb.isAdmin"
-                                        data-disabled
-                                    ></oui-action-menu-item>
-                                    <oui-action-menu-item
-                                        data-ng-if="privateDb.isAdmin"
-                                        data-text="{{::privateDb.name}}"
-                                        data-href="#/configuration/private_database/{{ privateDb.name }}"
-                                    >
-                                    </oui-action-menu-item>
-                                </div>
-                            </oui-action-menu>
                         </oui-tile-description>
                         <oui-action-menu data-compact data-align="end">
+                            <div
+                                data-ng-if="!hosting.sqlPriveInfo.hasPrivateDatabaseToActivate"
+                                data-ng-repeat="privateDb in privateDatabases track by privateDb.name"
+                            >
+                                <oui-action-menu-item data-disabled>
+                                    <span data-ng-bind="privateDb.name"></span>
+                                </oui-action-menu-item>
+                                <oui-action-menu-item
+                                    data-ng-if="privateDb.isAdmin"
+                                    data-href="#/configuration/private_database/{{ privateDb.name }}"
```

Comment:
if possible, use ui-router states, in the routing

Reply:
Same story, it's a DTrun, it has been running for a whole week, I already did too much 😅